prediction_market_agent/omen.py

```diff
@@ -468,5 +473,45 @@ def omen_sell_outcome_tx(
         check_tx_receipt(withdraw_receipt)
 
 
+def omen_claim_winnings(
```

Comment:
In other repositories, they had a standard to append `_tx` to the function that consumes gas in the background; could we have it, too?
```suggestion
def omen_claim_winnings_tx(
```